### PR TITLE
normalize docs in become plugins

### PR DESCRIPTION
**Pull request by russoz**
Normalize doc blocks for plugins

##### ISSUE TYPE

- Docs Pull Request

##### COMPONENT NAME

plugins/become/doas.py
plugins/become/dzdo.py
plugins/become/ksu.py
plugins/become/machinectl.py
plugins/become/pbrun.py
plugins/become/pfexec.py
plugins/become/pmrun.py
plugins/become/run0.py
plugins/become/sesu.py
plugins/become/sudosu.py

### CODE REVIEW

File: plugins/become/pmrun.py [hunk 1]
```diff
@@ -5,60 +5,60 @@
...
+      - name: ansible_become_pass
+      - name: ansible_pmrun_pass
+    env:
+      - name: ANSIBLE_BECOME_PASS
+      - name: ANSIBLE_PMRUN_PASS
+    ini:
+      - section: pmrun_become_plugin
+        key: password
+notes:
+  - This plugin ignores the become_user supplied and uses pmrun's own configuration to select the user.
```
Comment (felixfontein) [inline plugins/become/pmrun.py:60]:
```suggestion
  - This plugin ignores the C(become_user) supplied and uses pmrun's own configuration to select the user.
```

**Comment by felixfontein** [comment]
This now has conflicts.

File: plugins/become/pmrun.py (hunk 1 above)
Comment (russoz) [reply plugins/become/pmrun.py]: done

**Review by felixfontein — APPROVED** [review]

**Comment by felixfontein** [comment]
@russoz thanks for your contribution!

**Pull request merged**